apps/core/webpagequeue.cpp

```diff
@@ -159,6 +159,18 @@ int WebPageQueue::tabId(uint32_t uniqueId) const
     return 0;
 }
 
+int WebPageQueue::tabUsedBefore() const
+{
+    if (m_queue.count() > 1) {
+        WebPageEntry *pageEntry = m_queue.at(1);
+        if (pageEntry) {
+            return pageEntry->tabId;
+        }
+        return 0;
```

Comment:
I think this extra return is not necessary here.

Reply:
Very true... Looks like a leftover. Will drop.